app/src/Components/ListTable/ListTableRowButtons.js

```diff
@@ -29,23 +32,34 @@ export const ListTableRowButtons = ({buttons, value}) => {
 }
 
 function _createButton(button, value) {
-  switch(button.function) {
+  switch(button.name) {
```

Comment:
I really don't like abusing a button name as a function identifier. Either rename it to button type or re-add the function (IMO)

Reply:
renamed 'name' to 'function' as we dont use name for icon buttons if the first place..